.github/workflows/bi-connectors.yml

```diff
@@ -42,13 +42,13 @@ jobs:
       working-directory: bi-connectors/PowerBIConnector
     - name: Upload Tableau JDBC connector
       if: steps.pack-tableau-jdbc.outcome == 'success'
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v4
```

Comment:
That's good we are moderning the CI flow, but is there any reason behind the jump of 3 major versions? (v1 -> v4)